package.json

```diff
@@ -175,11 +176,11 @@
     "i18n:compile": "tasks/compileI18n.js",
     "i18n:build": "npm run i18n:extract && npm run i18n:update && npm run i18n:compile",
     "posti18n:build": "./tasks/createI18nCache.js && ./tasks/commitI18n.sh",
-    "pretty": "prettier --write $(find src/app -type f -name '*.js')",
-    "precommit": "./tasks/metrics.sh",
+    "pretty": "prettier --write \"src/app/**/*.js\" ",
+    "precommit": "node ./tasks/metrics.js",
     "create": "appComponent",
```

Comment:
this dependency is not cross-platform too, it uses spawn with bash argument internally

Reply:
Ho you're right :+1: 
But once the docker will be ready it won't be an issue anymore.